### DISCUSSION TITLE
Internal error RDP to Scaleset VM

### DISCUSSION THREAD

**Discussion by mgajownik** [AzureCommunityPL/group#263] [Pomoc]
Cześć macie może problemy lub mieliście z dostaniem się RDP lub Bastion do VM w scalesecie? Pomaga tylko restart VM. Moge sie zalogowac raz jak się rozłacze i chcę kolejny raz to mam Internal error accures ... to co w google sprawdziłem, chyba, ze pominąłem coś?

**Comment by smereczynski** [comment]
Jaka wersja WS? 2004?
